docs/examples/tunnel-with-reverse-proxy/manifests/ingress-nginx/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: ingress-nginx
+
+resources:
+  - resources/namespace.yaml
+
+helmCharts:
```

Comment:
Do people do this? Kustomize with helm charts? I was not aware, thanks!

Reply:
It's something I do, I've seen some use in the wild but it isn't super common AFAIK. It's a really nice way imo to make a declarative and (mostly) tool agnostic deployment for helm charts without something heavyweight like ArgoCD/Flux or something relatively uncommon like some of the third party helm wrappers. Also makes it portable across orchestrators as it prevents login